app/constants/autocomplete.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+export const AT_MENTION_REGEX = /\B(@([^@\r\n\s]*))$/i;
+
+export const AT_MENTION_SEARCH_REGEX = /\bfrom:\s*(\S*)$/i;
+
+export const CHANNEL_MENTION_REGEX = /\B(~([^~\r\n]*))$/i;
```

Comment:
Same here for [this line](https://github.com/mattermost/mattermost-mobile/pull/988/files#diff-d8e1907524aa62484d09c898587f4b91R49)

Reply:
I would prefer to keep the regex as is now, this way I'm sure they are working and we could change it later, how that sounds?